pkg/services/ngalert/remote/alertmanager_test.go

```diff
@@ -63,11 +63,12 @@ func TestNewAlertmanager(t *testing.T) {
 	for _, test := range tests {
```

Comment:
The only change in this file is replacing the `1` int values for the value set in the config struct.